app/src/main/res/layout/fragment_create_account.xml

```diff
@@ -22,60 +22,48 @@
         <ImageView
             android:id="@+id/logoImageView"
             android:layout_width="wrap_content"
-            android:layout_height="169dp"
```

Comment:
Can you set the weight on the logo image to 0.5 and set paddingBottom to 10dp on the LinearLayout. That will just push the text fields and button down to the bottom of the screen when makes better use of the background image.